online-fishing-store/src/app/pages/order-page/order-page.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { ButtonLabel } from 'src/app/enums/button-label-enum';
+import { User } from 'src/app/interfaces/user';
+import { UserService } from 'src/app/services/users/user.service';
+import { COUNTRIES } from '../profile-page/personal-data/countries';
+import { NotificationService } from 'src/app/services/notification/notification.service';
+import { ReportMessage } from 'src/app/enums/report-message-enum';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-order-page',
+  templateUrl: './order-page.component.html',
+  styleUrls: ['./order-page.component.scss']
+})
+export class OrderPageComponent implements OnInit {
+  public user: User;
+  public countries: string[] = COUNTRIES;
+  public headerValue: string = 'информация о доставке';
+  public checkoutButtonTitle: string = ButtonLabel.CHECKOUT;
+  public form: FormGroup;
+
+  constructor(
+    public userService: UserService,
+    private router: Router,
+    private notificationService: NotificationService,
+    private formBuilder: FormBuilder,
+  ) { }
+
+  ngOnInit(): void {
+    this.getUser();
+    this.form = this.formBuilder.group({
+      firstName: [this.user.firstName, [Validators.required]],
+      lastName: [this.user.lastName, [Validators.required]],
+      country: [this.user.country, [Validators.required]],
+      address: [this.user.address, [Validators.required]],
+      phoneNumber: [this.user.phoneNumber, [Validators.required, Validators.pattern('[0-9]{3} [0-9]{2} [0-9]{7}')]],
```

Comment:
Create utils folder in app and in this filder create form-validators.ts file. And into it create function phoneNumberPattern which returns this pattern. It should be reusable for other components